index.html

```diff
@@ -2,7 +2,7 @@
 <head>
     <title>Leaflet Draw Documentation</title>
     <script>
-        location = "http://leaflet.github.io/Leaflet.draw/docs/leaflet-draw-latest.html";
```

Comment:
This wasn't supposed to be in the PR, I created the PR before I pushed that. How can I remove it again?